test/plugin/constants.rb

```diff
@@ -186,8 +186,8 @@ module Constants
     detect_subservice false
   ).freeze
 
-  DISABLE_SPLIT_LOGS_BY_TAG_CONFIG = %(
-    split_logs_by_tag false
+  SPLIT_LOGS_BY_TAG_ENABLE_CONFIG = %(
```

Comment:
Hmm, the other `ENABLE` config constants are inconsistent in whether `ENABLE_` is a prefix or not. We should probably make them all use it as a prefix...

Reply:
I noticed this as well. Seems the only one that does not follow this pattern is `ENABLE_METADATA_AGENT_CONFIG` though. And that config itself is called `enable_metadata_agent`. So I left it unchanged. Do we want to change the config name to `ENABLE_METADATA_AGENT_ENABLE_CONFIG` or `ENABLE_METADATA_AGENT_TRUE_CONFIG`?

Reply:
I think `ENABLE_METADATA_AGENT_CONFIG` and `DISABLE_METADATA_AGENT_CONFIG` are reasonable here. So I would advocate for `ENABLE_SPLIT_LOGS_BY_TAG_CONFIG`, `ENABLE_PROMETHEUS_CONFIG`, etc.

Reply:
Updated.